browser-switch/src/test/java/com/braintreepayments/api/BrowserSwitchClientUnitTest.java

```diff
@@ -170,21 +178,51 @@ public void start_whenNoReturnUrlSchemeSet_throwsFailure() {
         BrowserSwitchOptions options = new BrowserSwitchOptions()
                 .requestCode(123)
                 .returnUrlScheme(null)
+                .appLinkUri(null)
                 .url(browserSwitchDestinationUrl)
                 .metadata(metadata);
         BrowserSwitchPendingRequest request = sut.start(componentActivity, options);
         assertTrue(request instanceof BrowserSwitchPendingRequest.Failure);
-        assertEquals("A returnUrlScheme is required.", ((BrowserSwitchPendingRequest.Failure) request).getCause().getMessage());
+        assertEquals("An appLinkUri or returnUrlScheme is required.", ((BrowserSwitchPendingRequest.Failure) request).getCause().getMessage());
     }
 
+
+    // TODO: FIX THIS
```

Comment:
I assume this should still be commented out?

Reply:
i ended up fixing the test case 👍